os/net/blemgr/bledev.c

```diff
@@ -33,9 +34,59 @@
 		}										\
 	} while (0)
 
+#define SCAN_INFO_BUFFER_SIZE 150
+static trble_scanned_device scan_info_buf[SCAN_INFO_BUFFER_SIZE] = { 0, };
+static int write_index = 0;
```

Comment:
How about renaming this and `read_index` with `g_`?

Reply:
I added prefix